src/Twitter/CreateTweetThroughApiCall.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Laminas\AutomaticReleases\Twitter;
+
+use Laminas\AutomaticReleases\Github\Event\MilestoneClosedEvent;
+use Laminas\AutomaticReleases\Twitter\Value\Tweet;
+use Laminas\Diactoros\Uri;
+use Laminas\Twitter\Response;
+use Laminas\Twitter\Twitter;
+use Psr\Http\Message\UriInterface;
+use stdClass;
+use Webmozart\Assert\Assert;
+
+use function sprintf;
+
+final class CreateTweetThroughApiCall
+{
+    private Twitter $twitter;
+
+    public function __construct(Twitter $twitter)
+    {
+        $this->twitter = $twitter;
+    }
+
+    public function __invoke(MilestoneClosedEvent $event): UriInterface
+    {
+        $tweet = Tweet::fromMilestoneClosedEvent($event);
+
+        $response = $this->statusesUpdate($tweet);
+
+        $responseObject = $response->toValue();
+        Assert::isInstanceOf($responseObject, stdClass::class);
```

Comment:
Urgh, is our API client this bad? :S

Reply:
We're doing the same as most API clients and just running json_decode() on the response payload, because it keeps us future-proof. 😄

Reply:
Did you just tell me "yes, our API client is that bad!"?

Reply:
At least it's not terrible, so it's good that it's bad, so at least it's not terrible :D

Reply:
Let's just say it's of roughly equal quality to every other API client out there. :stuck_out_tongue: